packages/logs/lib/es/client.ts

```diff
@@ -11,3 +12,42 @@ export const client = new Client({
         password: envs.NANGO_LOGS_ES_PWD! // ggignore
     }
 });
+
+function withCircuitBreaker(target: Client): Client {
+    const circuitBreaker = new CircuitBreaker({
+        healthCheck: async () => {
+            try {
+                await target.cluster.health();
```

Comment:
There might be a more specific way to check the ES cluster is ready for Nango logs. wdty @bodinsamuel ?